guides/common/modules/con_content-view-scenarios.adoc

```diff
@@ -8,40 +8,63 @@ It is not recommended to associate hosts directly with the Library as it prevent
 Instead, create a lifecycle environment path that suits your content workflow.
 The following scenarios are common:
 
-* *A single lifecycle environment* – content from Library is promoted directly to the production stage.
+* *A single lifecycle environment* {endash} content from Library is promoted directly to the production stage.
 This approach limits the complexity but still allows for testing the content within the Library before making it available to hosts.
 +
-image::lifecycle-path-basic-satellite.png[A single lifecycle environment]
+ifdef::foreman-el,foreman-deb,katello[]
+image::common/lifecycle-path-basic.svg[A single lifecycle environment]
+endif::[]
+ifdef::orcharhino[]
+// TODO: Add image
+endif::[]
+ifdef::satellite[]
+image::common/lifecycle-path-basic-satellite.png[A single lifecycle environment]
+endif::[]
 
-* *A single lifecycle environment path* – both operating system and applications content is promoted through the same path.
-The path can consist of several stages (for example *Development*, *QA*, *Production*), which enables thorough testing but requires additional effort.
+* *A single lifecycle environment path* {endash} both operating system and applications content is promoted through the same path.
+The path can consist of several stages, for example, *Development*, *QA*, *Production*, which enables thorough testing but requires additional effort.
```

Comment:
I think the parentheses were much easier to read. If you want to avoid parentheses at all costs, how about two sentences?

```suggestion
The path can consist of several stages, which enables thorough testing but requires additional effort.
For example, you can use  the *Development*, *QA*, and *Production* stages.
```